AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1918,6 +1925,7 @@ - (void)setPreferredFrameSize:(CGSize)preferredFrameSize
   ASDN::MutexLocker l(_propertyLock);
   if (! CGSizeEqualToSize(_preferredFrameSize, preferredFrameSize)) {
     _preferredFrameSize = preferredFrameSize;
+    self.sizeRange = ASRelativeSizeRangeMake(ASRelativeSizeMakeWithCGSize(_preferredFrameSize), ASRelativeSizeMakeWithCGSize(_preferredFrameSize));
```

Comment:
What if this value is set directly? Should we definitely override size range, or should it be considered an error to set both of these properties? I filed a ship task number 72 to track clarifying and hopefully unifying these APIs.  @nguyenhuy - not a priority, but in 2 weeks or so it would be great to have your input on where you see the right direction for these APIs, and we can do the implementation for you with our SF team - want to see you focused on the key priorities you have coming up!